JWTDecode.podspec

```
@@ -19,5 +19,5 @@ Pod::Spec.new do |s|
   s.visionos.deployment_target = '1.0'
 
   s.source_files = 'JWTDecode/*.swift'
-  s.swift_versions = ['5.9']
+  s.swift_versions = ['6.0.2', '6.0.3', '6.1']
```

```suggestion
  s.swift_versions = ['6.0', '6.1', '6.2']
```
Any particular reason for specifying `6.0.2` and `6.0.3` here?